dev/com.ibm.ws.security.jwt/resources/com/ibm/ws/security/jwt/internal/resources/JWTMessages.nlsprops

```diff
@@ -361,3 +361,24 @@ ERROR_BUILDING_SIGNED_JWE.useraction=Check the error that is included in the mes
 KEY_MANAGEMENT_KEY_NOT_FOUND=CWWKS6061E: The [{0}] JWT builder cannot create a JSON Web Encryption (JWE) token because a key management key cannot be found. They key management key alias is [{1}], and the configured trust store is [{2}].
 KEY_MANAGEMENT_KEY_NOT_FOUND.explanation=A key management key is a key that is used to encrypt the Content Encryption Key of a JWE token. A key management key must be specified so the JWT builder can create JWE tokens. A key management key can be specified by using the key alias and configured trust store values that are specified in the message or by using the JwtBuilder API.
 KEY_MANAGEMENT_KEY_NOT_FOUND.useraction=Verify that a key with the alias that is specified in the message exists in the trust store that is specified in the message. If the JwtBuilder API is used, ensure that a key management key is provided.
+
+# Do not translate "MicroProfile Config", "mp.jwt.verify.publickey.location"
+# 0=mp.jwt.decrypt.key.location
+DECRYPT_KEY_LOCATION_INLINE_KEY=CWWKS6062E: The value for the [{0}] MicroProfile Config property appears to include a private key string, which is not a supported configuration.
+DECRYPT_KEY_LOCATION_INLINE_KEY.explanation=The MicroProfile JWT feature does not support specifying private key strings in the MicroProfile Config property that is specified in the message. The value of this property must be a relative path or URL.
+DECRYPT_KEY_LOCATION_INLINE_KEY.useraction=Use a relative path or URL as the value for the MicroProfile Config property that is specified in the message. The location values that are supported are the same as the mp.jwt.verify.publickey.location property.
+
+# Do not translate "mp.jwt.decrypt.key.location", "keyManagementKeyAlias", "Content Encryption Key"
+# 0=JWT consumer config ID
+JWS_REQUIRED_BUT_TOKEN_NOT_JWS=CWWKS6063E: The token in the request is not in JSON Web Signature (JWS) format, but the [{0}] JWT consumer only accepts tokens that are in JWS format.
+JWS_REQUIRED_BUT_TOKEN_NOT_JWS.explanation=The token in the request might be malformed or might be in JSON Web Encryption (JWE) format. The MicroProfile JWT specification requires that only tokens in JWS format must be accepted if the JWE decryption key location is not configured.
```

Comment:
Is there a reason it says "must be" instead of "are"?

`The MicroProfile JWT specification requires that only tokens in JWS format are accepted if the JWE decryption key location is not configured.`

or

`The MicroProfile JWT specification requires that tokens must be in JWS format if the JWE decryption key location is not configured.`


It could be even more concise if the user doesn't need to care about the spec in order to fix the problem:

 `Tokens must be in JWS format if the JWE decryption key location is not configured.`

Reply:
My thought was to mention the spec so that the admin knows that it's not a limitation that our product has put on the request, but rather the specification itself. I'm perfectly fine making the change, though - I don't think mentioning the spec is really necessary.